ui.js

```diff
@@ -0,0 +1,254 @@
+import * as TTT from "./tic-tac-toe.js";
+
+const game = new TTT.TicTacToe();
+
+const circleGame = document.querySelectorAll(".little-circle");
+const crossGame = document.querySelectorAll(".little-cross");
+
+const bigCircle = "big-circle";
+const bigCross = "big-cross";
+const WinBox = "win";
+
+const resetButton = document.querySelector("button");
+const playerSign = document.querySelectorAll(".player");
+const boxes = document.querySelectorAll(".boxes");
+
+const textWindow = document.querySelector("#text-game-container");
+const textWindowText = document.querySelector("#text-screen");
+
+let closeTextWindow = () => {
```

Comment:
This should be a `const` (the open function also).

But in this case, it would be more easier to write
```
function closeTextWindow(){
    ///blabla
}
```

Reply:
This way makes me grow accustomed to the arrow functions.